### PR TITLE
[thread-cert] fix case 6.1.9 random fail

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: Cert_6_1_09_EDSynchronization.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
```
Comment (jwhui) [inline Cert_6_1_09_EDSynchronization.json:1]: Is adding this file (and pcap file below) intentional?

**Comment by jwhui** [comment]
I noticed that 6.1.9 still failed with this PR.

File: Cert_6_1_09_EDSynchronization.json (hunk 1 above)
Comment (FiveDimensions) [reply Cert_6_1_09_EDSynchronization.json]: sorry, wrong commit, deleted

**Comment by FiveDimensions** [comment]
> I noticed that 6.1.9 still failed with this PR.

yes, works locally, looking into it

**Review by jwhui — APPROVED** [review]

File: tests/scripts/thread-cert/Cert_6_1_09_EDSynchronization.py [hunk 2]
```diff
@@ -58,7 +58,8 @@ class Cert_6_1_9_EDSynchronization(thread_cert.TestCase):
         ED: {
             'name': 'ED',
             'panid': 0xface,
-            'whitelist': [LEADER, ROUTER1, ROUTER2, ROUTER3]
+            'router_upgrade_threshold': 0,
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_6_1_09_EDSynchronization.py:61]: should use `routereligible disable`?

**Pull request merged**